README.md

```diff
@@ -67,6 +67,8 @@ In comparison, storm-graphite (this project) provides a
 [GraphiteMetricsConsumer](src/main/java/com/verisign/storm/metrics/GraphiteMetricsConsumer.java) that reports these
 metrics to a [Graphite](https://github.com/graphite-project/graphite-web) server instead of writing to a file.
 
+For large Storm clusters emitting a high volume of metrics, we've included a [KafkaReporter](src/main/java/com/verisign/storm/metrics/reporters/KafkaReporter.java) that reports metrics to a configurable [Kafka](https://kafka.apache.org/) topic.
```

Comment:
What about mentioning Kafka support also at the very top of the README, where it's easier to spot?

Like:

> An Apache Storm IMetricsConsumer (source) implementation that forwards Storm's built-in metrics to a Graphite server for real-time graphing, visualization, and operational dashboards.  We also provide the option to forward these metrics to Apache Kafka instead of Graphite, which may be preferable when your Storm clusters are large.